src/main/java/org/jboss/modules/ref/package-info.java

```diff
@@ -23,5 +23,4 @@
  * @deprecated Use {@link java.lang.ref.Cleaner} or one of the reference types from a support library such as
  *      {@code io.smallrye.common:smallrye-common-ref} instead.
  */
-@Deprecated
-package org.jboss.modules.ref;
\ No newline at end of file
+package org.jboss.modules.ref;
```

Comment:
Did you mean to un-deprecate this?

Reply:
No, when building the 2.x branch locally I'm getting build failure if @deprecated annotation is used on package name @dmlloyd :

---
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /home/opalka/git/redhat/PUBLIC/jboss-modules/src/main/java/org/jboss/modules/ref/package-info.java:[27,1] modifier deprecated not allowed here
[INFO] 1 error
---

Reply:
I guess it must be due to having an earlier JDK target. That's fine then!